Sources/XCTest/XCTestCase.swift

```diff
@@ -92,6 +99,8 @@ extension XCTestCase {
 
                 XCTPrint("Test Case '\(fullName)' started.")
 
+                observationCenter.testCaseWillStart(testCase)
```

Comment:
Here's a crazy idea: could we define an `XCTestObserver` that we use internally to call `XCTPrint()`? That is, could we use our own observation infrastructure to print the default test output?

I'm not sure off the top of my head whether Apple XCTest does this, but I think it'd be wild! :sunglasses: